server/src/controllers/Events/resolver.ts

```diff
@@ -740,6 +740,7 @@ ${venueDetails}`;
 
   // TODO: This will need a real GraphQL return type (AFAIK you have to return
   // an object type)
+  @Authorized(Permission.EventEdit)
```

Comment:
```suggestion
  @Authorized(Permission.EventSendInvite)
```
This deserves its own permission, to be honest.